core-services/prow/02_config/_plugins.yaml

```diff
@@ -1279,6 +1289,30 @@ owners:
   - approved
   - lgtm
 plugins:
+  cloud-bulldozer/plow:
+  - assign
+  - blunderbuss
+  - blockade
+  - bugzilla
+  - cat
+  - dog
+  - heart
+  - golint
+  - goose
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - override
+  - pony
+  - retitle
+  - shrug
+  - sigmention
+  - size
+  - skip
+  - trigger
+  - verify-owners
```

Comment:
You don't need this hunk if you only need periodics now.

Reply:
@petr-muller he doesn't need any changes to this plugins file, right?

Reply:
@sallyom correct, no plugins should be needed for this use case

Reply:
Ok, I have removed all changes at the _plugins.yaml file